app/src/cambodia/java/org/eyeseetea/malariacare/monitor/utils/SurveyMonitor.java

```diff
@@ -21,39 +21,39 @@
 import org.eyeseetea.malariacare.database.model.Survey;
 import org.eyeseetea.malariacare.database.model.Value;
 
-import java.util.Date;
-import java.util.List;
-
 /**
  * Decorator that tells if a survey has specific info
  * Created by arrizabalaga on 26/02/16.
  */
-public class SurveyMonitor {
+public class SurveyMonitor extends BaseSurveyMonitor{
 
+    public SurveyMonitor(Survey survey){
+        super(survey);
+    }
     /**
      * Id of first question (positive, negative, not tested)
      */
-    private final static Long ID_QUESTION_RDT=1l;
+    protected final static Long ID_QUESTION_RDT=1l;
```

Comment:
@idelcano  something minor but do we have to specify the field as 'protected'? I though default scope is almost the same as protected...